### PR TITLE
Improving billing logging sink docs

### PR DESCRIPTION
**Pull request by chrisst**
To clarify https://github.com/terraform-providers/terraform-provider-google/issues/1703

### CODE REVIEW

File: website/docs/r/logging_billing_account_sink.html.markdown [hunk 1]
```diff
@@ -13,7 +13,8 @@ Manages a billing account logging sink. For more information see
 [Exporting Logs in the API](https://cloud.google.com/logging/docs/api/tasks/exporting-logs).
 
 Note that you must have the "Logs Configuration Writer" IAM role (`roles/logging.configWriter`)
-granted to the credentials used with terraform.
+granted on [the billing account not the project](https://cloud.google.com/billing/docs/how-to/billing-access) to
```
[the remark this answers is not in the document]
Comment (chrisst) [reply website/docs/r/logging_billing_account_sink.html.markdown]: I was feeling a bit awkward about the 'not the project syntax', this is much cleaner.

File: website/docs/r/logging_billing_account_sink.html.markdown [hunk 2]
```diff
@@ -13,7 +13,9 @@ Manages a billing account logging sink. For more information see
 [Exporting Logs in the API](https://cloud.google.com/logging/docs/api/tasks/exporting-logs).
 
 Note that you must have the "Logs Configuration Writer" IAM role (`roles/logging.configWriter`)
-granted to the credentials used with terraform.
+[granted on the billing account](https://cloud.google.com/billing/reference/rest/v1/billingAccounts/getIamPolicy) to
```
Comment (rileykarson) [inline website/docs/r/logging_billing_account_sink.html.markdown]: Grammar change: what do you think about moving the subject around to make this sentence less passive? Something along the lines of:

"The credentials you are using with Terraform must have the "Logs Configuration Writer" IAM role (`roles/logging.configWriter`) [granted on the billing account](https://cloud.google.com/billing/reference/rest/v1/billingAccounts/getIamPolicy) that is used with this sink.

File: website/docs/r/logging_billing_account_sink.html.markdown [hunk 3]
```diff
@@ -13,7 +13,9 @@ Manages a billing account logging sink. For more information see
...
+the credentials used with terraform. [IAM roles granted on a billing account are separate from the
```
Comment (rileykarson) [inline website/docs/r/logging_billing_account_sink.html.markdown]: Nit: link text can be cut down to "IAM roles granted on a billing account" to better represent what we are linking here

File: website/docs/r/logging_billing_account_sink.html.markdown [hunk 4]
```diff
@@ -12,8 +12,10 @@ Manages a billing account logging sink. For more information see
 [the official documentation](https://cloud.google.com/logging/docs/) and
 [Exporting Logs in the API](https://cloud.google.com/logging/docs/api/tasks/exporting-logs).
 
-Note that you must have the "Logs Configuration Writer" IAM role (`roles/logging.configWriter`)
-granted to the credentials used with terraform.
+~> **Note** You must have the "Logs Configuration Writer" IAM role (`roles/logging.configWriter`)
+[granted on the billing account](https://cloud.google.com/billing/reference/rest/v1/billingAccounts/getIamPolicy) to
+the credentials used with terraform. [IAM roles granted on a billing account](https://cloud.google.com/billing/docs/how-to/billing-access) are separate from the
```
Comment (rileykarson) [inline website/docs/r/logging_billing_account_sink.html.markdown]: s/terraform/Terraform/

**Pull request merged**